.gitmodules

```diff
@@ -1,6 +1,3 @@
 [submodule "hs_docker_base"]
 	path = hs_docker_base
 	url = https://github.com/hydroshare/hs_docker_base.git
```

Comment:
Instead of chaining repo and dockerfiles, consolidate into a single dockerfile in the hydroshare/hydroshare repo. Simplification, configuration consistency.